mathematical-companion/main.tex

```diff
@@ -82,7 +83,8 @@ \subsection{Polynomial Rings}
 \textbf{unit}, if and only if it is nonzero constant.
 
 If whenever a polynomial $r$ is written as the product of two polynomials $r=pq$,
-either $p$ or $q$ is a unit (i.e.~degree 0), then we say $r$ is \textbf{irreducible}.
+either $p$ or $q$ is a unit (i.e.~degree 0), then we say $r$ is \textbf{reducible}.
```

Comment:
The original reads as correct to me, though perhaps awkwardly phrased.

Reply:
Ah, I think you're right. Lemme just rewrite this line.